sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1315 @@
+    "    primary_metric=\"NormalizedRootMeanSquaredError\",\n",
+    "    n_cross_validations=3,\n",
+    "    enable_model_explainability=True,\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    ")\n",
+    "\n",
+    "# Limits are all optional\n",
+    "forecasting_job.set_limits(\n",
+    "    timeout_minutes=600,\n",
+    "    trial_timeout_minutes=20,\n",
+    "    max_trials=max_trials,\n",
+    "    # max_concurrent_trials = 4,\n",
+    "    # max_cores_per_trial: -1,\n",
+    "    enable_early_termination=True,\n",
+    ")\n",
+    "# Featurization properties are optional\n",
+    "# forecasting_job.set_featurization(# drop_columns=[\"not_needed_column\"], # Optional\n",
+    "#                                   # enable_dnn_featurization=True         # Enable if there are text columns\n",
+    "#                                     )"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Specialized properties for Time Series Forecasting training\n",
+    "forecasting_job.set_forecast_settings(\n",
+    "    time_column_name=TIME_COLNAME,\n",
+    "    forecast_horizon=FORECAST_HORIZON,\n",
+    "    target_lags=TARGET_LAGS,\n",
+    "    time_series_id_column_names=TIME_SERIES_ID_COLNAMES,\n",
+    "    use_stl=STL_TYPE,\n",
+    "    # ADDITIONAL FORECASTING TRAINING PARAMS ---\n",
+    "    # frequency=\"H\",\n",
+    "    # time_series_id_column_names=[\"tid1\", \"tid2\", \"tid2\"],\n",
+    "    # short_series_handling_config=ShortSeriesHandlingConfiguration.DROP,\n",
+    "    # use_stl=\"season\",\n",
+    "    # seasonality=3,\n",
+    "    # target_rolling_window_size=4,\n",
+    ")\n",
+    "\n",
+    "# Training properties are optional\n",
+    "forecasting_job.set_training(blocked_training_algorithms=[\"ExtremeRandomTrees\"])"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### 4.2 Train the AutoML model"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Using the `MLClient` created earlier, we will execute the following commands to train the model."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Submit the AutoML job\n",
+    "returned_job = ml_client.jobs.create_or_update(\n",
+    "    forecasting_job\n",
+    ")  # submit the job to the backend\n",
+    "\n",
+    "print(f\"Created job: {returned_job}\")"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Wait until AutoML training runs are finished\n",
+    "ml_client.jobs.stream(returned_job.name)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 5. Retrieve the Best Trial (Best Model's trial/run)\n",
```

Comment:
The V1 notebook has a section where best run details is fetched. Can we discuss this?

Reply:
I have added it back after removing it as I need this info : f"azureml://jobs/{best_run.info.run_id}/outputs/artifacts/outputs/model.pkl" to register the model so that we can use it for running forecast on test dataset.